picohttp/demoserver.h

```diff
@@ -72,7 +72,7 @@ int picoquic_h09_server_callback(picoquic_cnx_t* cnx,
  * according to the ALPN selected by the client
  */
 
-size_t picoquic_demo_server_callback_select_alpn(picoquic_quic_t* quic, ptls_iovec_t* list, size_t count);
+size_t picoquic_demo_server_callback_select_alpn(picoquic_quic_t* quic, picoquic_iovec_t* list, size_t count);
 
```

Comment:
OK, but the function call in the implementation should be set to the V2 API.

Reply:
We have an issue here. The API `picoquic_demo_server_callback_select_alpn` is used in `demo_server_test` in `h3zerotest.c`, and in `picowt_baton_test_one` in `webtransport_test.c`. We need to update these tests to call `picoquic_set_alpn_select_fn_v2`.